randovania/layout/preset_migration.py

```diff
@@ -1239,6 +1239,18 @@ def _migrate_v94(preset: dict) -> dict:
     return preset
 
 
+def _migrate_v95(preset: dict) -> dict:
```

Comment:
Remove, corruption is development

Reply:
corruption already has preset migrations, so we're committed to continuing to migrate. do not remove